tap-snapshots/test/v8-to-istanbul.js.test.cjs

```diff
@@ -2871,7 +3154,7 @@ Object {
     "37": 0,
     "38": 1,
     "39": 1,
-    "4": 1,
+    "4": 2,
```

Comment:
This snapshot has been fixed because the original coverage was wrong. "4" same as "5" have count 2 because they are inside the range in `multi-source-map.js`
![image](https://user-images.githubusercontent.com/160981/146622983-36551859-dfa4-42d3-bd76-9f9cffd1f277.png)